terra-env/main.tf

```diff
@@ -93,7 +93,7 @@ module "workspace_manager" {
 }
 
 module "crl_janitor" {
-  source = "github.com/broadinstitute/terraform-ap-modules.git//crl-janitor?ref=crl-janitor-0.2.1"
+  source = "github.com/broadinstitute/terraform-ap-modules.git//crl-janitor?ref=crl-janitor-0.2.2"
```

Comment:
Also for my education: what does this version bump do? Does it need to be part of this PR? Will you have to create a release tag manually after this PR goes in? (And do we have this process documented somewhere?)

Reply:
The version bump tells terra-env to use the newer version of crl-janitor(because we update that module). Also after this PR get merged, I need to bump terra-env version in ap-deployments depo:
https://github.com/broadinstitute/terraform-ap-deployments/pull/96/files

The workflow is:  (1) Make changes in this repo, and update version for the deployments we changed(crl-janitor in this case). (2)After the PR merged, **manually** create two tags crl-janitor 0.2.2 and terra-env-0.3.6. Push the two tags. 
(3) Bump terra-env version in ap-deployment repo to use terra-env-0.3.6 
There are some description: 
https://github.com/broadinstitute/terraform-ap-deployments#2-create-pull-requests

Also, to verify this works, during (1) step, I need to let it refer to my local branch:
https://github.com/broadinstitute/terraform-ap-modules/pull/67/commits/eb287aa40e496296cb08f4c330a0489cd6f6efdc, 
same thing in ap-deloyment repo:
https://github.com/broadinstitute/terraform-ap-deployments/pull/96/commits/31321a2a1f50f27faac2cf96601859d198ae3104
Once plan works, do the step 2 and 3. 

The process is little bit complicated, let me know if it's not clear.